CoreHelp/src/au/gov/asd/tac/constellation/help/ConstellationHelpDisplayer.java

```diff
@@ -54,6 +54,7 @@
     private static final Logger LOGGER = Logger.getLogger(ConstellationHelpDisplayer.class.getName());
 
     private static final String OFFICIAL_CONSTELLATION_WEBSITE = "https://www.constellation-app.com/help";
+    private static final String NEWLINE = "\n";
```

Comment:
Instead of this I would use `SeparatorConstants.NEWLINE`

Reply:
`SeparatorConstants` is in `Core Utilities` and this is in `Core Help`.  `Core Utilities` has a dependancy on `Core Help`, so adding project Core Utilities as dependency to Core Help would introduce cyclic dependency.

Reply:
Ah right. I guess this can probably be left as is then